src/pages/TeamSettings/Service-Accounts-Table.vue

```diff
@@ -236,7 +236,9 @@ export default {
       }
     },
     keys: {
-      query: require('@/graphql/Tokens/api-keys.gql'),
+      query() {
+        return require('@/graphql/Tokens/api-keys.js').default(this.isCloud)
```

Comment:
Again this page is only accessed by cloud so I think we can simplify this and run it as a standard query.